src/billing/BillingAccount/BillingAccountControls.tsx

```diff
@@ -197,7 +198,7 @@ export const BillingAccountControls = (props: BillingAccountControlsProps) => {
                 <ButtonPrimary
                   onClick={() => {
                     setShowBillingRemovalModal(false);
-                    removeBillingAccount(selectedBilling).then(reloadBillingProject);
+                    removeBillingAccount().then(reloadBillingProject);
```

Comment:
The method doesn't declare any arguments and just references `selectedBilling` directly.